src/tests/suite/daos_rebuild_ec.c

```diff
@@ -66,6 +66,14 @@ rebuild_ec_internal(void **state, uint16_t oclass, int kill_data_nr,
 		rebuild_pools_ranks(&arg, 1, &extra_kill_ranks[2], 2, false);
 	}
 
+	if (write_type == PARTIAL_UPDATE)
+		verify_ec_partial(&req, arg->index, 0);
+	else if (write_type == FULL_UPDATE)
+		verify_ec_full(&req, arg->index, 0);
+	else if (write_type == FULL_PARTIAL_UPDATE)
+		verify_ec_full_partial(&req, arg->index, 0);
+	else if (write_type == PARTIAL_FULL_UPDATE)
+		verify_ec_full(&req, arg->index, 0);
```

Comment:
hmm, just curious, why add this back?  daos_obj_verify() does not work for EC object?

Reply:
daos_obj_verify() only verify EC data consistency, but cannot verify the data is same as expected, no?